doc/endpoint/security.md

```diff
@@ -52,7 +52,7 @@ Optional and multiple authentication inputs have some additional rules as to how
 ## Limiting request body length
 
 *Supported backends*: 
-This feature is available for backends based on http4s, jdkhttp, Netty, and Play. More backends will be added in the near future.
+This feature is available for backends based on http4s, jdkhttp, Netty, pekko-http, akka-http, and Play. More backends will be added in the near future.
```

Comment:
maybe it's easier now to enumerate those which we don't support? ;) this can come together with flipping the default test flag from `false` to `true` and explicitly marking the backends which don't support the feature